### DISCUSSION TITLE
Is the HTML audio element blacklisted?

### DISCUSSION THREAD

**Discussion by gdippe** [datenstrom/community#513] [Ask a question] [answered]
Hello,
This might be a classical case of RTFM but I've spent a few hours trying to solve this which I think is a reasonable effort :)

I'd like to make an mp3-file available for playing on the same page where I have pictures but despite several tries nothing seems to work.

Example html.
`    <audio controls>
      <source src="/media/audio/akulla.mp3" type="audio/mpeg">
    </audio>
`
Is there a workaround if the audio-tag is blacklisted (not whitelisted)?
Cheers
Günther

**Comment by markseuffert** [comment, marked as answer]
Oh, this is probably my fault. 

The `controls` attribute was recently added to the HTML whitelist, then the virus outbreak happened and I didn't have much time for software maintenance. I made a new release now with a new HTML whitelist. In case you're curious, all safe elements/attributes are defined in the core in function `normaliseData()`. Can you update your website and test it again? 

I am really sorry about the time you had to spent on troubleshooting.

**Comment by gdippe** [comment]
No problem! I really appreciate all the time and effort you and other people here spend on making Yellow such a web publishing gem!
I haven't either had the time to maintain my website but now I figured I have to revive it.
Again, thank you for your quick response and help!
I very much appreciate it!
And stay safe!

**Comment by gdippe** [comment]
Now it works :)
The short recording can be found here.
https://gdfoto.se/blog/akulla

**Comment by markseuffert** [comment]
Thank you for testing and feedback.
